doc/cmsgemos.cfg

```diff
@@ -588,7 +588,8 @@ WARN_LOGFILE           =
 # directories like "/usr/src/myproject". Separate the files or directories 
 # with spaces.
 
-INPUT                  = $(BUILD_HOME)/$(GEM_OS_PROJECT)
+INPUT                  = $(BUILD_HOME)/$(GEM_OS_PROJECT) \
+                         $(BUILD_HOME)/$(GEM_OS_PROJECT)/gemonlinedb/gemonlinedb.md
```

Comment:
What is the result of this?
For the documentation, there will be two parts:
* the API (code comments generated by `doxygen`)
* the usage manual (done from the `md`/`rst` files in a nice readthedocs/sphinx style)
They will not be pushed to the same place, so keeping their creation distinct was the original goal

Reply:
It creates [this page](http://homepage.iihe.ac.be/~lmoureau/GEM/cmsgemos/doc/build/html/md_gemonlinedb_gemonlinedb.html). I intend to use it as a technical high-level overview overview of the package (something like [this](http://doc.qt.io/qt-5/application-windows.html)), and would like to include links to detailed class documentation.

If there's a way to reference Doxygen output from Sphinx, I'd be glad to use it. For now `make doc` doesn't run Sphinx...

Reply:
OK, definitely fits the part I mentioned above.

> If there's a way to reference Doxygen output from Sphinx, I'd be glad to use it. For now make doc doesn't run Sphinx...

Not currently, but on the internal (@jsturdy's brain) to-do list for once core features are fully implemented :-)
What you can assume will exist is a site structure like:
```
cmsgemos/sw/release/Maj.Min/api/latest (symlink to latest built version of MajMin)
cmsgemos/sw/release/Maj.Min/api/Maj.Min.Patch
cmsgemos/sw/release/Maj.Min/repo/<arch>/yum-repo-structure
```

What is not yet decided is where the global "user" docs will go, wether to a `cmsgemdaq` aggregate site, or a package specific site (I slightly lean towards the latter, with the former hosting a more overview type set of info)
```
cmsgemos/doc/<sphinx/rtd style doc site, versioned?>
```
This allows the user docs to be more dynamic, while allowing full API access to the versioned `doxygen` generated info based on releases.

So if (in your spare time) you want to think on how this can all work together, please feel free,
You can then open an issue when you have a concrete proposal and we can further discuss/refine.
Until then, I'd prefer that the `doxygen` config remain unchanged

Reply:
I'll include include top-level API docs into a docstring for `gem::onlinedb` once I have an idea of what the final API will look like.